assets/scss/base/_lists.scss

```diff
@@ -14,6 +25,26 @@ ul, nav ul {
 	}
 }
 
+/*
+Numbered list
+
+An ordered list with numbers where the parent element uses the modifier `.list--numbered` and the child elements use the class `.numbered`
+
+Markup:
```

Comment:
@adamconder That's not to say I don't believe you. But it'd be good to know whether we should raise an issue against this repo as a refactor job or just get the team that's using `div` and `p` to review their semantics.

How many instances did you find?